<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-05-07T13:52:15 IST" name="Suite" finished-at="2022-05-07T13:52:40 IST" duration-ms="24632">
    <groups>
    </groups>
    <test started-at="2022-05-07T13:52:15 IST" name="LoginTest" finished-at="2022-05-07T13:52:40 IST" duration-ms="24632">
      <class name="com.internetbank.testcases.TC_001_login">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:com.internetbank.testcases.TC_001_login@9f116cc]" started-at="2022-05-07T13:52:15 IST" name="setUp" finished-at="2022-05-07T13:52:34 IST" duration-ms="18109" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="logIn()[pri:1, instance:com.internetbank.testcases.TC_001_login@9f116cc]" started-at="2022-05-07T13:52:34 IST" name="logIn" finished-at="2022-05-07T13:52:38 IST" duration-ms="3947" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method signature="getTitle()[pri:2, instance:com.internetbank.testcases.TC_001_login@9f116cc]" started-at="2022-05-07T13:52:38 IST" name="getTitle" finished-at="2022-05-07T13:52:38 IST" duration-ms="11" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Guru99 Bank Manager HomePage] but found [ Guru99 Bank Manager HomePage ]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Guru99 Bank Manager HomePage] but found [ Guru99 Bank Manager HomePage ]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.internetbank.testcases.TC_001_login.getTitle(TC_001_login.java:37)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getTitle -->
        <test-method signature="logOut()[pri:3, instance:com.internetbank.testcases.TC_001_login@9f116cc]" started-at="2022-05-07T13:52:38 IST" name="logOut" finished-at="2022-05-07T13:52:40 IST" duration-ms="1556" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.internetbank.testcases.TC_001_login@9f116cc]" started-at="2022-05-07T13:52:40 IST" name="tearDown" finished-at="2022-05-07T13:52:40 IST" duration-ms="134" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.internetbank.testcases.TC_001_login -->
    </test> <!-- LoginTest -->
  </suite> <!-- Suite -->
</testng-results>
